src/hooks/useKeyboardShortcut/useKeyboardShortcut.types.ts

```diff
@@ -0,0 +1,103 @@
+import type { CharacterKey, KeyCode, modifierKeys } from './keyCodes.js';
+
+export type Key = Exclude<KeyCode, keyof typeof modifierKeys>;
+
+type Letters =
```

Comment:
Letters could be a lot broader than just the characters of the alphabet, maybe consider renaming to `EnglishAlphabetCharacters` 😉